app/models/contact/integrations/chatwoot/generate_conversation_link.rb

```diff
@@ -0,0 +1,33 @@
+class Contact::Integrations::Chatwoot::GenerateConversationLink
+  def initialize(contact)
+    @contact = contact
+  end
+
+  def call
+    chatwoot = fetch_chatwoot_for_account
+    chatwoot_contact_id = @contact.additional_attributes['chatwoot_id']
+
+    return { error: 'no_chatwoot_or_id' } unless chatwoot && chatwoot_contact_id
+
+    conversation_id = fetch_conversation_id(chatwoot, chatwoot_contact_id)
+    return { error: 'no_conversation' } unless conversation_id
+
+    { ok: build_conversation_url(chatwoot, conversation_id) }
+  end
+
+  private
+
+  def fetch_chatwoot_for_account
+    Apps::Chatwoot.first
+  end
+
+  def fetch_conversation_id(chatwoot, chatwoot_contact_id)
+    conversations = Accounts::Apps::Chatwoots::GetConversations.call(chatwoot, chatwoot_contact_id)
+    conversations.dig(:ok, 0, 'id')
+  end
+
+  def build_conversation_url(chatwoot, conversation_id)
+    conversation_path = "app/accounts/#{chatwoot.chatwoot_account_id}/conversations/#{conversation_id}"
```

Comment:
```suggestion
    conversation_path = "/app/accounts/#{chatwoot.chatwoot_account_id}/conversations/#{conversation_id}"
```

To achando que precisa adicionar essa / aqui.
Pelo teste que eu fiz abriu assim
![image](https://github.com/user-attachments/assets/b900bf46-2707-4977-8467-3d74e6573bea)

Reply:
Tem razão.

Reply:
Tem um problema nesse commit. 
Eu testei uma ocasião onde a pessoa cadastra a url do chatwoot no woofed tendo no final da url o caratere `/`.
Quando isso acontece, a url q será gerada ficará assim:  http://localhost:3000//app/accounts/2/conversations/8.
Até aí tudo bem, mas o problema é q quando a gente abre a pagina do chatwoot com essa url, ele n redireciona para a pagina de contato e sim para essa pagina: `http://localhost:3000/app/accounts/2/dashboard`

Reply:
Obs:
No formulário para cadastrar o chatwoot, o placeholder está assim:

![image](https://github.com/user-attachments/assets/8fc8c04f-c906-4623-8347-9d6e7946d5e5)

Dando a entender q o usuário tem q colocar a barra no final da url.